migrations/applied_migrations/ContactEmail.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
@shaileshp0110 Please remove the .sh files we do need to have them in the main branch.